tower-test/Cargo.toml

```diff
@@ -8,23 +8,22 @@ name = "tower-test"
 #   - README.md
 # - Update CHANGELOG.md.
 # - Create "v0.1.x" git tag.
-version = "0.1.0"
+version = "0.3.0-alpha.1"
 authors = ["Tower Maintainers <team@tower-rs.com>"]
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/tower-rs/tower"
 homepage = "https://github.com/tower-rs/tower"
-documentation = "https://docs.rs/tower-test/0.1.0"
+documentation = "https://docs.rs/tower-test/0.3.0-alpha.1"
 description = """
 Utilities for writing client and server `Service` tests.
 """
 categories = ["asynchronous", "network-programming"]
 edition = "2018"
 
 [dependencies]
-futures = "0.1.26"
-tokio-sync = "0.1.3"
-tower-service = "0.2.0"
-
-[dev-dependencies]
-tower = { version = "0.1.0", path = "../tower" }
+futures-util-preview = "0.3.0-alpha.18"
+futures-executor-preview = "0.3.0-alpha.18"
```

Comment:
We should drop this dependency and use https://docs.rs/tokio-test/0.2.0-alpha.4/tokio_test/ which contains mock tasks.